### ISSUE TITLE
MathJax still not rendering even with --javascript-delay

### ISSUE THREAD

**Issue by jxu** [wkhtmltopdf/wkhtmltopdf#5240]
**wkhtmltopdf version(s) affected**: 0.12.6

**OS information**
Ubuntu 20.04

**Description**
MathJax doesn't render even with sufficient javascript delay

**How to reproduce**
`wkhtmltopdf --javascript-delay 10000 https://projecteuler.net/problem=805 805.pdf`

**Expected behavior**
MathJax renders

**Comment by Sicos1977** [comment]
https://github.com/Sicos1977/ChromeHtmlToPdf

**Comment by jxu** [comment]
@Sicos1977 yes I saw. I would use chromium but half the time it spits out a blank PDF due to a known bug. Maybe I will retry spam chromium until it generates a proper PDF.

**Comment by arjunsuresh** [comment]
Which mathjax version are you using? It works fine for me with mathjax 2.7 but not with mathjax 3

**Comment by arjunsuresh** [comment]
https://github.com/wkhtmltopdf/wkhtmltopdf/issues/5030

**Comment by jxu** [comment]
> Which mathjax version are you using? It works fine for me with mathjax 2.7 but not with mathjax 3

whichever is used by the project euler site

**Comment by arjunsuresh** [comment]
yes. They use [Mathjax3](view-source:https://projecteuler.net/problem=6). Change it to mathax 2.7 and wkhtmltopdf should work. [These](https://github.com/GATEOverflow/GO-PDFs/releases) equation heavy PDFs are made using wkhtmltopdf and Mathjax.

**Comment by dalies** [comment]
Any update on this?
